### PR TITLE
initial cluster setup

### PR DESCRIPTION
**Pull request by folksgl**
- Initial Consul cluster setup
- Add description and intro content

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# Due to this, attempting to 'consul acl bootstrap' too early will result in
+# the command erroring, citing "The ACL system is currently in legacy mode."
+sleep 90
+
+# Begin ACL bootstrapping process
+cf ssh -N -L 8500:localhost:8080 consul-0 &
+consul0_ssh_pid=$!
+sleep 5 # Allow ssh connection to be established
+
+#master_token=$(consul acl bootstrap | sed -n '2 p' | awk '{ print $2 }')
```
Comment (dzaslavskiy) [inline bootstrap.sh:99]: master token is being passed into script so this is commented out for future refrence?

File: certificates/ca/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+The CA certificates, `consul-agent-ca-*.pem`, contain the public keys necessary
+to validate Consul certificates and therefore must be distributed to every node
+that runs a consul agent.
+
+Note that three CAs are required in order to ensure that the dev/test/prod
+datacenters are not capable of with one another. By signing the
```
Comment (dzaslavskiy) [inline certificates/ca/README.md:6]: missing word

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+server at a time, and operators should ensure that each server comes back in
+sync before starting the next.
+
+# Steps to build a new cluster (bootstrapping)
+These steps will loosely follow the 
+[Consul Deployment Guide](https://learn.hashicorp.com/tutorials/consul/deployment-guide?in=consul/production-deploy)
+
+1. [Install Consul](https://www.consul.io/docs/install#install-consul) on your local machine
+2. Ensure you are targeting the correct environment with `cf target -s <environment>`
+3. Run [boostrap.sh](bootstrap.sh) to deploy the cluster to the currently targeted CF environment
```
Comment (dzaslavskiy) [inline README.md:23]: where do i get a master token?

File: bootstrap.sh (hunk 1 above)
Comment (folksgl) [reply bootstrap.sh:99]: Ahh no, this was the initial way I was bootstrapping the cluster, before I realized we can eliminate the step by using a known starting secret for the master token. Deleting.

File: README.md (hunk 3 above)
Comment (folksgl) [reply README.md:23]: The master token is a secret, and will need to be stored as such. So only our internal dev team will have access.

File: server-config.hcl [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+cert_file = "./certificates/server/dc1-server-consul-${CONSUL_SERVER_INDEX}-${ENVIRONMENT_NAME}.pem"
+key_file = "./certificates/server/dc1-server-consul-${CONSUL_SERVER_INDEX}-key-${ENVIRONMENT_NAME}.pem"
+verify_incoming = true
+verify_outgoing = true
+verify_server_hostname = true
+auto_encrypt = {
+  allow_tls = true
+}
+tls_min_version = "tls12"
+#auto_config = {
```
Comment (dzaslavskiy) [inline server-config.hcl:50]: This commented block is needed?

Comment (folksgl) [reply server-config.hcl:50]: I'm unsure what the best way to move forward is on client agent connectivity. This comment block is certainly not _needed_ for the base setup here though, no. Removing.

**Review by dzaslavskiy — APPROVED** [review]

**Pull request merged**